lib/patreon/utils/jsonapi/url_util.rb

```diff
@@ -10,7 +10,7 @@ def self.build_url(url, includes=nil, fields=nil, count=nil, cursor=nil)
           params = parsed_url.query ? Rack::Utils.parse_query(parsed_url.query) : {}
           params['include'] = joined_or_null(includes) if includes
           fields.each do |name, val|
-            params["fields[#{name}]"] = val
+            params["fields[#{name}]"] = val.kind_of?(Array) ? val.join(',') : val
```

Comment:
Checking the class of a value is a Python-ism; instead, prefer checking if a method exists on a value.
```suggestion
            params["fields[#{name}]"] = val.respond_to?(:join) ? val.join(',') : val
```